### PR TITLE
Remove unused imports in FBE.

### LINKED ISSUE #2243: Remove unused imports in mimir/precog
Enable "-Ywarn-unused-import".

### PR DESCRIPTION
**Pull request by alissapajer**
Closes #2243.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -510,6 +514,7 @@ lazy val niflheim = project.setup.noArtifacts
 
 lazy val yggdrasil = project.setup
   .dependsOn(blueeyes % BothScopes, precog % BothScopes, niflheim % BothScopes)
+  .withWarnings
```
Comment (alissapajer) [inline build.sbt]: Added `withWarnings` here.

File: project/Precog.scala [hunk 2]
```diff
@@ -7,13 +7,13 @@ object Build {
...
-    "-Ywarn-unused-import",
-    "-Ywarn-numeric-widen",
-    "-Ywarn-value-discard")
+    //"-g:vars",
+    //"-deprecation",
+    //"-unchecked",
+    //"-Ywarn-value-discard",
+    //"-Ywarn-numeric-widen",
+    //"-Ywarn-unused",
+    "-Ywarn-unused-import")
```
Comment (alissapajer) [inline project/Precog.scala]: Here is where we can incrementally enable warnings, and also enable `"-Xfatal-warnings"`.

**Comment by djspiewak** [comment]
Is this us?

```
[info] ExamplesSpec
[error]   x support providers delegating services to addons
[error]    the value is false (AccessControlSpec.scala:311)
```

**Comment by alissapajer** [comment]
@djspiewak yeah that's a new one. It's in the blueeyes project. We should make an issue. That looks like all it printed out about the error.

**Comment by alissapajer** [comment]
Issue made #2296

**Comment by alissapajer** [comment]
Build is green except for marklogic_json.

**Pull request merged**